modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -233,6 +231,58 @@ function FieldBase({
 			type === 'radio');
 	const showPopover = fieldName === 'inputMaskFormat';
 
+	useLayoutEffect(() => {
+		console.log(type);
+		focusRequiredElement();
+	});
+
+	function focusRequiredElement() {
+		let errorInput = null;
+
+		const errorContainer = document.querySelector('.form-feedback-group');
+
+		if (errorContainer) {
+			switch (type) {
+				case 'select':
+					errorInput = errorContainer.parentElement.querySelector(
+						'.form-control.results-chosen.select-field-trigger'
+					);
+					break;
+				case 'date':
+				case 'image':
+				case 'document_library':
+				case 'color':
+					errorInput = errorContainer.parentElement.querySelector(
+						'.form-control'
+					);
+					break;
+				case 'radio':
+					errorInput = errorContainer.parentElement.querySelector(
+						'.custom-control-input'
+					);
+					break;
+				case 'checkbox_multiple':
+					errorInput =
+						errorContainer.parentElement.querySelector(
+							'.simple-toggle-switch.toggle-switch'
+						) ||
+						errorContainer.parentElement.querySelector(
+							'.custom-control-input'
+						);
+					break;
+				case 'grid':
+					errorInput = errorContainer.parentElement.querySelector(
+						'.custom-control-input.form-builder-grid-field'
+					);
+					break;
+			}
+			if (errorInput) {
+				errorInput.focus();
+			}
+			return;
+		}
+	}
+
```

Comment:
@joaovictoralvesmartins, apparently, this is all the code you need to fix the issue, in light of this can you please create a new pull with a single commit having this code only?